src/fontra/client/core/glyph-controller.js

```diff
@@ -616,7 +616,7 @@ export class StaticGlyphController {
     const selectionRects = [];
     if (pointIndices.length) {
       const path = filterPathByPointIndices(this.instance.path, pointIndices);
-      if (path) {
+      if (path.getBounds()) {
```

Comment:
You're calling `path.getBounds()` twice, which is wasteful. Please use a variable to prevent this.

Reply:
Thanks. done.